src/plugins/curator/components/create-teaser-modal/delegate.js

```diff
@@ -9,6 +9,16 @@ class Delegate extends AbstractDelegate {
       formName: formNames.CREATE_TEASER,
     }, dependencies);
   }
+
+  /**
+   * @inheritdoc
+   */
+  handleSubmit(data, formDispatch, formProps) {
```

Comment:
@rsumilang this stuff actually goes into the subscribers, e.g. https://github.com/triniti/cms-js/blob/e6c65009aebf1240df1888367b97bbbb6def657a/src/plugins/curator/services/AssetTeaserSubscriber.js#L50